test/e2e/http/actors.test.ts

```diff
@@ -380,4 +382,24 @@ describe("http/actors", () => {
       expect(res2).toEqual(123);
     });
   });
+
+  describe("setStateWithTTL", () => {
+    it("should be able to set actor state with TTL", async () => {
+      const builder = new ActorProxyBuilder<DemoActorTTLStateInterface>(DemoActorTTLStateImpl, client);
+      const actor = builder.build(ActorId.createRandomId());
+      await actor.init();
+
+      const data = { name: "John", age: 30 };
+      const ttl = 5000; // 5 seconds
```

Comment:
Isn't TTL configured in seconds? The comment says so 
>    * @param ttl The time-to-live (in seconds) for the state, after which it will expire and be removed.

Also this test times out in e2e, are you able to run it locally?